src/card/Card.tsx

```diff
@@ -0,0 +1,21 @@
+import "./_card.scss";
+
+import React from "react";
+import classNames from "classnames";
+
+import Header from "./header/Header";
+import Body from "./body/Body";
```

Comment:
```suggestion
import CardHeader from "./header/CardHeader";
import CardBody from "./body/CardBody";
```
Let's rename this files and components